### PR TITLE
Add 'copy' button to wikis

### LINKED ISSUE #423: Add a button to copy patches from an existing wiki into the create form
If you have a wiki with a few patches that have since been updated, you want to be able to copy the patch list into the create form to simplify creating an updated wiki.

### PR DESCRIPTION
**Pull request by edg2s**
Fixes #423

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -306,6 +309,17 @@
...
 
+	$actions = [];
+	if ( $canDelete ) {
+		$actions[] = '<a href="delete.php?wiki=' . $wiki . '">Delete</a>';
+	}
+	if ( $canCreate ) {
+		$patchList = array_map( static function ( $data ) {
+			return htmlspecialchars( $data['r'] );
+		}, $wikiData['patchList'] );
+		$actions[] = '<a class="copyWiki" href="?patches=' . implode( ',', $patchList ) . '&branch=' . htmlspecialchars( $wikiData['branch'] ) . '">Copy</a>';
```
Comment (edg2s) [inline index.php:320]: Adds noJS support for middle clicks

File: index.php [hunk 2]
```diff
@@ -359,15 +373,16 @@
...
-<script src="js/PatchSelectWidget.js"></script>
-<script src="js/index.js"></script>
-<?php
 echo '<script>
+window.pd = window.pd || {};
 pd.wikiPatches = ' . json_encode( $wikiPatches ) . ';
 pd.config = ' . json_encode( [
 	'phabricatorUrl' => $config['phabricatorUrl']
 ] ) . ';
 </script>';
```
Comment (edg2s) [inline index.php:382]: Needed to change the load order as we can set PatchSelectWidget value in the constructor if values loaded from query string. wikiPatches needs to be available when the PSW changes value.

File: js/PatchSelectWidget.js [hunk 3]
```diff
@@ -12,22 +12,23 @@ $( function () {
 			pd.patchIndex[ patchKey ].push( wiki );
 		}
 	}
-} );
+}() );
```
Comment (edg2s) [inline js/PatchSelectWidget.js:15]: Loads patchIndex sooner (along with load order).

File: js/PatchSelectWidget.js [hunk 4]
```diff
@@ -12,22 +12,23 @@ $( function () {
...
 window.PatchSelectWidget = function PatchSelectWidget( config ) {
 	var widget = this;
 
-	// Parent constructor
-	window.PatchSelectWidget.super.call( this, $.extend( {
-		allowArbitrary: true,
-		allowDisplayInvalidTags: true
-	}, config ) );
-
 	this.$formInput = $( '<input>' ).attr( {
```
Comment (edg2s) [inline js/PatchSelectWidget.js:20]: $formInput needs to be available immediately if the parent constructor changes the value

**Review by MatmaRex — APPROVED** [review]

**Pull request merged**